<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMapWindow</class>
 <widget class="QMainWindow" name="NewMapWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>657</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QGridLayout" name="mainGridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item row="0" column="0">
       <widget class="QTabWidget" name="mapObjectsWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>295</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>295</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="documentMode">
         <bool>true</bool>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="movable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Cell params</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget1">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>266</width>
            <height>400</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="horizontalSpacing">
            <number>6</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="11" column="0">
            <widget class="QLineEdit" name="earfcnDl">
             <property name="placeholderText">
              <string>:earfcnDl</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Channel number for the central DL frequency</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QLineEdit" name="transmitPower">
             <property name="placeholderText">
              <string>:transmitPower</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="site">
             <property name="placeholderText">
              <string>:site</string>
             </property>
            </widget>
           </item>
           <item row="15" column="0">
            <widget class="QLineEdit" name="ulNoiseAndInterference">
             <property name="placeholderText">
              <string>:ulNoiseAndInterference</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLineEdit" name="position_X">
             <property name="placeholderText">
              <string>:position_X</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="cell">
             <property name="placeholderText">
              <string>:cell</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>uplink N+I in dBm</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Cell position, coordinate X:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string> Site instance name:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Cell position, coordinate Y:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string> Transmit power in dBm</string>
             </property>
            </widget>
           </item>
           <item row="16" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pbSetCellParams">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Accept</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbRestoreCell">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Restore default</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <widget class="QLineEdit" name="pci">
             <property name="placeholderText">
              <string>:pci</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Physical cell identifier:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLineEdit" name="position_Y">
             <property name="placeholderText">
              <string>:position_Y</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Cell instance name:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Center params</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>265</width>
            <height>207</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="4" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>West boundary of area</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="northBoundary">
             <property name="placeholderText">
              <string>:northBoundary</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLineEdit" name="eastBoundary">
             <property name="placeholderText">
              <string>:eastBoundary</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLineEdit" name="westBoundary">
             <property name="placeholderText">
              <string>:westBoundary</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>South boundary of area</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>North boundary of area</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>East boundary of area</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="southBoundary">
             <property name="placeholderText">
              <string>:southBoundary</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pbSetCenterParams">
               <property name="text">
                <string>Accept</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbRestoreCenter">
               <property name="text">
                <string>Restore default </string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Handover params</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>15</y>
            <width>266</width>
            <height>207</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="horizontalSpacing">
            <number>6</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>South boundary of area</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>North boundary of area</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="northHandovernoundary">
             <property name="placeholderText">
              <string>northBoundary</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLineEdit" name="eastHandovernoundary">
             <property name="placeholderText">
              <string>:eastBoundary</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>East boundary of area</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>West boundary of area</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="soutHandovernoundary">
             <property name="placeholderText">
              <string>:southBoundary</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLineEdit" name="westHandovernoundary">
             <property name="placeholderText">
              <string>:westBoundary</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="pbsetHandoverParams">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Accept</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbRestoreHandover">
               <property name="text">
                <string>Restore default </string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
